Hazelcast.Net/Hazelcast.Client.Connection/ClientConnection.cs

```diff
@@ -49,6 +50,7 @@ internal sealed class ClientConnection
         private readonly BlockingCollection<ISocketWritable> _writeQueue = new BlockingCollection<ISocketWritable>();
 
         private volatile Socket _clientSocket;
+        private volatile Stream _stream;
```

Comment:
this shouldn't need to be volatile, but instead readonly as it's only created once.